draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -143,6 +143,14 @@ The protocol described here bases its design on the following protocol requireme
 
 # Selection of DNS API Server {#selection}
 
+Configuration, discovery, and updating of the URI Template {{RFC6570}} is done out of band from this
+protocol. Note that configuration might be manual (such as a user typing URI Templates in
+a user interface for "options") or automatic (such as URI Templates being supplied in
+responses from DHCP or similar protocols). DNS API Servers MAY support more
+than one URI. This allows the different endpoints to have different
+properties such as different authentication requirements or service
+level guarantees.
+
 A DNS API client uses configuration to select the URI, and thus the
 DNS API server, that is to be used for resolution. [RFC2818] defines how HTTPS
```

Comment:
Is this sentence duplicative now?

Reply:
I don't think so. It extends what was said above.